base/applications/mspaint/lang/tr-TR.rc

```diff
@@ -73,7 +74,7 @@ BEGIN
             END
             MENUITEM SEPARATOR
             MENUITEM "K&ılavuzu Göster\tCtrl+G", IDM_VIEWSHOWGRID
-            MENUITEM "K&üçük Resim Göster", IDM_VIEWSHOWMINIATURE
+            MENUITEM "Kü&çük Resim Göster", IDM_VIEWSHOWMINIATURE
```

Comment:
did you double-check that change on turkish 2k3sp2? Can you send a pic of the Microsoft menu?